### DISCUSSION TITLE
MiBox does not turn on

### DISCUSSION THREAD

**Discussion by brunopiras** [PRProd/HA-Firemote#167] [General]
Hello there, 

MiBox TV it does not turn on when I press the on/off button, but if I turn on when the remote, all works fine and I can turn off.

Any piece of advice?

**Comment by PRProd** [comment]
@brunopiras - Greetings!  Thanks for joining us!

Unfortunately, this is an issue that I not able to solve.  One of our discussion contributors @moh-Brent has wrestled with this as well, but so far we cannot come up with a working solution.  Home Assistant integrations are also unable to turn on a Xiaomi MiBox that has been powered off.  

The way Firemote works is that it uses Home Assistant integrations in the background, while providing a nice looking card to add to your dashboard.  Unfortunately, since Home Assistant integrations can't handle a power-on event, neither can Firemote.  It appears that the MiBox will only accept a power-on command from the IR remote right now.

**Reply by PRProd** [reply]
I did see a suggestion somewhere that said you could use a smart plug to power it on.  Firemote is set up in such a way that you could easily change the power button to toggle a smart plug instead.

**Reply by moh-Brent** [reply]
A smart plug would do the job but takes a long time to boot, I don't have the patience.
I have been looking at using an ESP32 and bluetooth to replicate the remote, but either I'm not clever enough, highly possible, or Wireshark can't see the commands

**Reply by PRProd** [reply]
I asked an AI (google Bard), and it didn't have any additional information on this topic either:
![image](https://github.com/PRProd/HA-Firemote/assets/25486599/73fc14b7-f612-4377-a661-301a839c42a9)
